mlflow/store/sqlalchemy_store.py

```diff
@@ -19,9 +19,36 @@
 
 
 class SqlAlchemyStore(AbstractStore):
+    """
+    SQLAlchemy compliant backend store for tracking meta data for MLflow entities. Currently
+    supported database types are ``mysql``, ``mssql``, ``sqlite``, and ``postgresql``. This store
+    interacts with SQL store using SQLAlchemy abstractions defined for MLflow entities.
+    :py:class:`mlflow.store.dbmodels.models.SqlExperiment`,
+    :py:class:`mlflow.store.dbmodels.models.SqlRun`,
+    :py:class:`mlflow.store.dbmodels.models.SqlTag`,
+    :py:class:`mlflow.store.dbmodels.models.SqlMetric`, and
+    :py:class:`mlflow.store.dbmodels.models.SqlParam`.
+
+    Run artifacts are stored in a separate location using artifact stores conforming to
+    :py:class:`mlflow.store.artifact_repo.ArtifactRepository`. Default artifact locations for
+    user experiments are stored in database along with meta data for with
+    :py:class:`mlflow.store.dbmodels.models.SqlExperiment`. Each run artifact location is recorded
+    in :py:class:`mlflow.store.dbmodels.models.SqlRun` and stored in backend DB.
+    """
```

Comment:
Could you use articles (a, the) consistently; either include or drop.

Default artifact locations for user experiments are stored in database  -> Default artifact locations for user experiments are stored in a database or Default artifact locations for user experiments are stored in the database 

stored in backend DB -> stored in a backend DB or stored in the backend DB